script/import_scripts/smf2.rb

```diff
@@ -336,16 +337,21 @@ def convert_message_body(body, attachments = [], **opts)
         end
       end
       if use_count.keys.length < attachments.select(&:present?).length
-        body << "\n\n---"
+        body = "#{body}\n\n---"
         attachments.each_with_index do |upload, num|
           if upload.present? && use_count[num] == (0)
-            body << ("\n\n" + get_upload_markdown(upload))
+            "#{body}\n\n#{get_upload_markdown(upload)}"
           end
         end
       end
     end
 
-    opts[:ignore_quotes] ? body : convert_quotes(body)
+    if opts[:ignore_quotes]
+      body
+    else
+      body = convert_quotes(body)
```

Comment:
Same here, I would move `convert_quotes` inside `convert_bbcode` and just have one call

```ruby
body = convert_bbcode(body)
```